nexus/deployment/src/planner.rs

```diff
@@ -83,7 +83,21 @@ impl<'a> Planner<'a> {
         // is fine.
         let mut sleds_ineligible_for_services = BTreeSet::new();
 
+        // TODO: this code will also need to _remove_ services from expunged
+        // (and decommissioned?) sleds.
+
```

Comment:
FWIW this is tracked on the Reconfigurator board so I don't think we need this.

Reply:
Removed, thanks.